phys2denoise/metrics/chest_belt.py

```diff
@@ -155,30 +193,67 @@ def env(resp, samplerate, window=10):
        young adults scanned at rest, including systematic changes and 'missed'
        deep breaths," Neuroimage, vol. 204, 2020.
     """
+
+    def _respiratory_pattern_variability(data, window):
```

Comment:
Now that you can either input a physio object or a numpy array, do we still need to duplicate code for this application?

Reply:
That's true the relevant line in `env()` could be changed as such:
```python
env_arr = (
        pd.Series(data.data)
        .rolling(window=window, center=True)
        .apply(respiratory_pattern_variability, args=(window,), kwargs=dict(return_physio=False))
    )
```
However we should also include support for `pandas.core.series.Series` types in the `load_physio` function for that to work, which also impacts the other module. Still, I think it'd be better to open an issue about it, because it complicates the PRs
Wdyty @smoia